components/CommentCard/CommentCard.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react";
+import CommentCard from "./CommentCard";
+
+const testComment1 = "sehr Lecker.";
+("This restaurant is very picturesque. The sorrounding is relaxing, which makes it a great place to spend lunch with your boyfriend/girlfriend but also with your friends or family. The hausgemachte Limonade is fantastic. I tried the lemon one and it's very fresh and well done. Despite being a bit costly, the portion is big and satisfying. I suggest to take the Antipasti. The only down is the service, it could have been better. There were not enough cutlery and napkins for everyone.Ich kann es nur Empfehlen.");
```

Comment:
The long comment is not part of the `testComment1` because of the colon. Do you need it and it's a mistake?